index.html

```diff
@@ -5,8 +5,54 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link rel="stylesheet" href="./styles/index.css">
 </head>
 <body>
+    <header>
+        <h1>Weather Report</h1>
+        <h3> ~  Love Mother Nature  ~ </h3>
+        <h2 id="cityNameDisplay">Seattle</h2>
```

Comment:
Recall that the heading tag represents hierarchy on a page so we should have h1, then h2, the h3